### ISSUE TITLE
[Test Failure][Unstable]The info in the “Preview Changes” window is not correct when trying to uninstall local packages from the existing solution

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#9487]
## Details about Problem
NuGet version: NuGet Client Dev\5.7.0.6595
VS Version: Master\30027.154
OS: Windows-10-Enterprise-RS5
## Detailed repro steps 
1. Opne an exist solution that already installed several embedded icon packages.
2. Open the solution PM UI and go to “Installed” tab.
3. Choose an embedded icon package and click “Uninstall” button.
## Expected
The info in the prompt “Preview Changes” window should be correct.
## Actual
The info in the prompt “Preview Changes” window is not correct as below screenshot. It try to install a package which had been installed on the project.
![PreviewChanges](https://user-images.githubusercontent.com/20632625/80495022-f2849c00-8999-11ea-8963-5db521bb2a0b.png)
## Notes
1. It also repro in the project PM UI.
2. Package “FwPackage” is a local created .NET Framework 4.8 package with embedded icon. Package “classlib.net48” is installed from devops source: https://pkgs.dev.azure.com/farhub/_packaging/farhub/nuget/v3/index.json.

**Comment by heng-liu** [comment]
I couldn't repro on VS 16.8 p1 with the following steps:
1. create a local  .NET Framework 4.8 package with embedded icon - package1
2. create a .NET Framework 4.8 classlib project
3. install package1 (tried both package reference and package config)
4. add source https://pkgs.dev.azure.com/farhub/_packaging/farhub/nuget/v3/index.json 
5. install classlib.net48 from the above source
6. go to "installed" tab
7. uninstall package1
8. check the "Preview Changes", everything is correct: only the package1 is under the uninstalling list.
@v-luzh can you help check if it's fixed on VS 16.7 p5? 
Pls let me know if the repro steps are not correct. Thanks!

**Comment by v-luzh** [comment]
Hi @heng-liu , It is an unstable issue. The repro ratio is 50%. I could repro it on 16.7 P5, 16.7 P6 and Master latest.

My repro steps:
1. Create a local .NET Framework 4.8 package with embedded icon - package1 **in old VS version** (e.g. 16.7 P4/Master old version)
2. Create a .NET Framework 4.8 classlib project
3. Install package1 (tried both package reference and package config)
4. Add source https://pkgs.dev.azure.com/farhub/_packaging/farhub/nuget/v3/index.json
5. Install classlib.net48 from the above source
6. **Update the VS to newer build (e.g. 16.7 P5/Master newer version)**
7. Reopen the exist solution (in step2) and open PM UI for solution
8. Go to "Installed" tab and uninstall the package1
9. Check the "Preview Changes"

**Comment by v-luzh** [comment]
The bug is fixed on Master\30404.108 + NuGet Client Dev\5.8.0.6765. Close this bug now, thanks.

**Comment by v-luzh** [comment]
It repro again during today's testing on Master\30406.155 + NuGet Client Dev\5.8.0.6768 as below. Reopen it now.
![image](https://user-images.githubusercontent.com/20632625/89633519-8e497080-d8d6-11ea-8c57-fb8d9fb594b5.png)

**Comment by v-luzh** [comment]
After investigating it, we found this issue is caused by bug 9311. VS crashed when trying installing a package with invalid embedded icon format. And when VS restarting, we didn't choose to recover the solution (clicked "Do Not Recover" button in below screenshot), and then it made the info in "Preview Changes" window incorrect. 
![image](https://user-images.githubusercontent.com/20632625/97157916-b77dec80-17b3-11eb-8d7e-f345a220f9b5.png)
With the fixing of VS crash issue (part of original actual results in bug 9311), this issue had been fixed. Close this issue now. Thanks.
